src/router.js

```diff
@@ -28,21 +29,25 @@ import WordDetail from './views/WordDetail.vue';
 const routes = [
   {
     path: '/',
+    name: 'feed',
+    component: FeedList,
+    beforeEnter(to, from, next) {
+      next(to.fullPath.startsWith('/#/') ? to.fullPath.slice(2) : undefined);
+    },
+  },
+  {
+    path: '/landing',
     name: 'landing',
     component: Landing,
+    beforeEnter(to, from, next) {
+      if (store.getters.isLoggedIn) next({ name: 'feed' });
```

Comment:
Is it important to have this redirect? I don't think that logged-in user may open this page accidentally, it contains a useful explanation of the service and has a link to the feed 🤷‍♀️ 

To access the store in redirection guard you may use a trick with `beforeRouteEnter` in-component guard:
https://router.vuejs.org/guide/advanced/navigation-guards.html#per-route-guard
```js
beforeRouteEnter (to, from, next) {
  next(vm => {
    if (vm.$store.getters.isLoggedIn) vm.$router.push({ name: 'feed' });
  })
}
```

Reply:
For me also it's not a problem a logged user to access the landing page, but it's like a common practice🤷🏻‍♂️ Now I'm thinking of removing this guard, still this trick is nice to know.